trunner/device.py

```diff
@@ -12,14 +12,25 @@
             '-nographic',
             '-monitor', 'none'
         ]
+    ),
+    'armv7a9-zynq7000-emu': (
+        'qemu-system-aarch64',
+        [
+            '-M', 'arm-generic-fdt-7series',
+            '-dtb', f'{PHRTOS_PROJECT_DIR}/scripts/zynq7000-zc702.dtb',
+            '-serial', 'null',
+            '-serial', 'mon:stdio',
+            '-device', f'loader,file={PHRTOS_PROJECT_DIR}/_boot/plo-armv7a9-zynq7000-qemu.img',
+            '-drive', f'file={PHRTOS_PROJECT_DIR}/_boot/flash-armv7a9-zynq7000.bin,if=mtd,format=raw,index=0'
+        ]
     )
 }
 
 
 class RunnerFactory:
     @staticmethod
     def create(target, serial, log=False):
-        if target == 'ia32-generic':
+        if target == 'ia32-generic' or target == 'armv7a9-zynq7000-emu':
```

Comment:
```python
if target in [ 'ia32-generic', 'armv7a9-zynq7000']:
```
or even specify `QEMU_TARGETS = [..]`

Should probably be upgraded soon to device-configs.